src/components/elements/Tabs/Default/TabButton.tsx

```diff
@@ -34,21 +34,22 @@ export const TabButton = ({
       className={classNames(
         className,
         `w-full items-center focus:outline-none disabled:text-neutral-900`,
-        "border",
+        "",
         item.done && "peer",
-        lastItem || selected ? "border-b" : "border-b-0",
+        lastItem || selected ? "" : "",
```

Comment:
This appears to be another NOOP, please remove.

Reply:
done